api-js/src/user/objects/user-personal.js

```diff
@@ -10,6 +10,19 @@ import { nodeInterface } from '../../node'
 
 const { CIPHER_KEY } = process.env
 
+export const decryptPhoneNumber = ({iv, tag, phoneNumber: encrypted}) => {
```

Comment:
Can you put this back where it was, it's the only location that its being used, so I'd prefer to keep it inside the resolver itself.

Reply:
I pulled this function out so we can use it to test for changed phone numbers like here: https://github.com/canada-ca/tracker/blob/5abafc0aac45cf9e39332fd2f6d62efc9506e996/api-js/src/user/mutations/__tests__/set-phone-number.test.js#L165

Reply:
okay, then remove it from the file, and create it, its own folder in the validators directory and add it to context, if you need help lmk.